projects/angular/src/layout/nav/_header.clarity.scss

```diff
@@ -516,12 +292,12 @@
 
       // Last Dropdown Menu adjustments so that the menu has some space between itself and the right end of the browser window
       &:last-child.bottom-right > .dropdown-menu {
-        right: variables.$clr_baselineRem_0_125;
+        right: mixins.baselinePx(3);
       }
 
       // Pull the dropdown menus of all dropdowns up to compensate for the height of the header.
       .dropdown-menu {
-        margin-top: -1 * variables.$clr_baselineRem_4px;
+        margin-top: calc(-1 * tokens.$cds-global-space-3);
```

Comment:
I think we had some discussion, but decided to keep it as it is, because it was leading to some effects.
I prefer to have it addressed in the refactor task, so I will add it there.